client/account-recovery/reset-password/reset-password-form/reset-option-set.jsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+
+const getResetOptionDisplayString = ( optionName, translate ) => {
+	switch ( optionName ) {
+		case 'primary':
+			return translate( 'main', {
+				comment: 'It is about which way a user wants to reset the password. e.g. main email, recovery email.',
+			} );
+		case 'secondary':
+			return translate( 'recovery', {
+				comment: 'It is about which way a user wants to reset the password. e.g. main email, recovery email.',
+			} );
+		default:
+			return '';
+	}
+};
+
+const ResetOptionSet = ( props ) => {
+	const {
+		email,
+		sms,
+		name,
+		disabled,
+		translate,
+		onOptionChanged,
+		selectedResetOption,
+	} = props;
+
+	const emailFieldValue = name + '_email';
+	const smsFieldValue = name + '_sms';
+	const optionDisplayName = getResetOptionDisplayString( name, translate );
+
+	return (
+		<div>
+			{ email && (
+				<FormLabel>
+					<FormRadio
+						className={ classnames( 'reset-password-form__email-option', name ) }
+						value={ emailFieldValue }
+						checked={ emailFieldValue === selectedResetOption }
+						disabled={ disabled }
+						onChange={ onOptionChanged } />
+					<span>
+						{ translate(
+							'Email a reset link to {{strong}}your %(optionName)s email address{{/strong}}',
+							{
+								components: { strong: <strong /> },
+								args: { optionName: optionDisplayName },
```

Comment:
Or just change the `getResetOptionDisplayString` function to something like

```
const getResetOptionDisplayString = ( optionName, translate ) => {
 	switch ( optionName ) {
 		case 'primary':
			return {
				email: translate(
					'Email a reset link to {{strong}}your main email address{{/strong}}',
					{ components: { strong: <strong /> } } ),
				sms: translate(
					'Send a reset code to {{strong}}your main phone{{/strong}}',
					{ components: { strong: <strong /> } } ),
			};
		case 'secondary':
			return {
				email: translate(
					'Email a reset link to {{strong}}your recovery email address{{/strong}}',
					{ components: { strong: <strong /> } } ),
				sms: translate(
					'Send a reset code to {{strong}}your recovery phone{{/strong}}',
					{ components: { strong: <strong /> } } ),
			};
		default:
			return '';
	}
}
```

You'll end up with 8 translated strings instead of 4 but at least you'll be guaranteed the perfect sentence for "Send a reset code to your main phone" in every language that its translated to.